pkg/sink/pulsar/config_test.go

```diff
@@ -116,3 +117,17 @@ func TestGetDefaultTopicName(t *testing.T) {
 	config, _ = NewPulsarConfig(sink, replicaConfig.Sink.PulsarConfig)
 	assert.Equal(t, config.GetDefaultTopicName(), "persistent://tenant/namespace/test-topic")
 }
+
+func TestNewPulsarConfigFailure(t *testing.T) {
```

Comment:
Added a UT for failure in PulsarConfig creation due to invalid pulsar schema  . Here, invalid pulsar schema is "pulsar+htp" instead of "pulsar+http"